scripts/linters/pylint_extensions.py

```diff
@@ -561,20 +665,114 @@ def check_arg_section_indentation(self, node):
         currently_in_freeform_section = False
         args_indentation = 0
         if node.doc:
-            # Process the docstring line by line.
+            current_docstring_section = None
+            in_description = False
+            args_indentation_in_spaces = 0
             for line in node.doc.splitlines():
-                line_with_no_leading_whitespaces = line.lstrip()
+                stripped_line = line.lstrip()
                 current_line_indentation = (
-                    len(line) - len(line_with_no_leading_whitespaces))
+                    len(line) - len(stripped_line))
                 parameter = re.search(
                     '^[^:]+:',
-                    line_with_no_leading_whitespaces)
+                    stripped_line)
                 # Check for empty lines and ignore them.
                 if len(line.strip()) == 0:
                     continue
+                # If line starts with Returns: , it is the header of a Returns
+                # subsection.
+                if stripped_line.startswith('Returns:'):
+                    current_docstring_section = (
+                        self.DOCSTRING_SECTION_RETURNS)
+                    in_freeform_section = False
+                    in_description = False
+                    args_indentation_in_spaces = current_line_indentation
+                # If line starts with Raises: , it is the header of a Raises
+                # subsection.
+                if stripped_line.startswith('Raises:'):
```

Comment:
Shouldn't these be elif? (This and below, the previous code had elif)

Reply:
Changed to elif